lokerpc/schema_test.go

```
@@ -289,3 +353,98 @@ func TestTypeSchema(t *testing.T) {
+			if !reflect.DeepEqual(got, tt.want) {
+				gotstr, _ := json.MarshalIndent(got, "", "  ")
+				wantstr, _ := json.MarshalIndent(tt.want, "", "  ")
+				t.Errorf("TypeSchema() = %s, want %s", gotstr, wantstr)
+			}
+		})
+	}
+}
+
+// selfEmbed embeds itself; promoting its fields must terminate.
+type selfEmbed struct {
+	*selfEmbed
```

golangci-lint (2.12.2)

[error] field selfEmbed is unused

(unused)